### PR TITLE
[Changed] Modify Dockerfile to work when using make docker commands

### PR DESCRIPTION
**Pull request by mgarcia0094**
### :tophat: What?
Modify Dockerfile to work fine when using make docker commands

### :thinking: Why?
When we use docker make commands, we are receiving this error:

image (3)

We have fixed this file to work fine :)

I also updated the list of code owners.
**The deleted users are due to not having write access to the repository.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (andrewgy8) [inline .github/CODEOWNERS:1]:
```suggestion
* @daniel-ruiz @EmilioCarrion @mgarcia0094 @AdamjGoddard
```
😜

Comment (mgarcia0094) [reply .github/CODEOWNERS:1]: I forgot to upload the last commit D:. Thank you!

**Review by daniel-ruiz — APPROVED** [review]
@mgarcia0094 please fix the errors in the CODEOWNERS file before merging. 
Very good job at fixing this error 👏

File: .github/CODEOWNERS
Comment (andrewgy8) [inline .github/CODEOWNERS:1]:
```suggestion
* @daniel-ruiz @EmilioCarrion @mgarcia0094 @AdamjGoddard
```

**Pull request merged**
